dlt/common/destination/reference.py

```diff
@@ -234,6 +252,11 @@ def complete_load(self, load_id: str) -> None:
         """Marks the load package with `load_id` as completed in the destination. Before such commit is done, the data with `load_id` is invalid."""
         pass
 
+    @abstractmethod
+    def clear_load(self, load_id: str) -> None:
```

Comment:
what about `complete_load` above? what's wrong with it? looks like code duplication everywhere

Reply:
this is to clear the load with a certain id, I use that in the tests, because the tests where using sql before. it is a unified interface for sql and not sql clients. we can do something else though if you like.

Reply:
OK I get it now, the docstrings still say it is to add a load_id, not remove. this is needed by just one test. in that case I'd add it to the test itself. or even provoke the error by mocking `complete_load` wiht exception during the test so it is not executed